test/EndToEnd/tests/InstallPackageTest.ps1

```diff
@@ -1800,7 +1800,7 @@ function Test-InstallPackageInstallsHighestPackageIfItIsReleaseWhenPreReleaseFla
 function Test-InstallingPackagesWorksInTurkishLocaleWhenPackageIdContainsLetterI
 {
     # Arrange
-    $p = New-ClassLibrary
```

Comment:
I remember having issues like this with some other end to end tests. 

Details here:

https://github.com/NuGet/NuGet.Client/pull/1664

LSL vs non LSL changed the compatibility check somehow.